docs/chapter4-payload-handoff-format.rst

```diff
@@ -29,9 +29,836 @@ architecture, either 32 or 64 bits.
+   #address-cells    R           u32          Specifies the number of <u32> cells to represent the
+                                              address in the reg property in children of root.
+   #size-cells       R           u32          Specifies the number of <u32> cells to represent the
+                                              size in the reg property in children of root.
+   ================= =========== ============ ======================================================
+
+
+4.2.2 Node: /options/upl-params (R)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+These are the generic parameters / settings of the UPL payload.
+
+.. tabularcolumns:: | p{3cm} p{0.75cm} p{2cm} p{9.5cm} |
+.. table:: Node: /options/upl-params
+
+   ================= =========== ============ ======================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================
+   Property          Usage       Value Type   Definition
+   ================= =========== ============ ======================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================
+   compatible        R           string       "upl"
+   boot-mode         O           string list  For supporting different Payload boot flows. These influence how the payload runs. Note that this is not intended to support multiple Payloads.
+
+                                              “normal” means booting with full functionality, from power off.
+                                              “fast” means fast boot with no configuration change or minimal functionality.
+                                              “full", "diag” means booting with full functionality plus diagnostics functions.
+                                              “default” means booting with “reset to default” flow (for example, boot from RTC battery power failure).
+                                              “s4” means resuming from ACPI S4 state (resume from disk).
+                                              “factory” means booting with manufacturing mode.
+                                              “s3” means resuming from ACPI S3 state (resume from memory).
```

Comment:
These look really bad in the resulting PDF. Isn't there a way to force a new line after each possible option?

Reply:
Done, made them into point forms.